installation/sql/postgresql/base.sql

```diff
@@ -228,7 +228,8 @@ INSERT INTO "#__extensions" ("package_id", "name", "type", "element", "folder",
 (0, 'mod_articles_category', 'module', 'mod_articles_category', '', 0, 1, 1, 0, 1, '', '', '', 0, 0),
 (0, 'mod_articles_categories', 'module', 'mod_articles_categories', '', 0, 1, 1, 0, 1, '', '', '', 0, 0),
 (0, 'mod_languages', 'module', 'mod_languages', '', 0, 1, 1, 0, 1, '', '', '', 0, 0),
-(0, 'mod_finder', 'module', 'mod_finder', '', 0, 1, 0, 0, 1, '', '', '', 0, 0);
+(0, 'mod_finder', 'module', 'mod_finder', '', 0, 1, 0, 0, 1, '', '', '', 0, 0),
+(0, 'mod_content', 'module', 'mod_content', '', 0, 1, 0, 0, 1, '', '', '');
```

Comment:
```suggestion
(0, 'mod_finder', 'module', 'mod_finder', '', 0, 1, 0, 0, 1, '', '', '', 0, 0),
(0, 'mod_content', 'module', 'mod_content', '', 0, 1, 0, 0, 1, '', '', '', 0, 0);
```
This is the reason why the installation fails on PostgreSQL and so system tests for PostgreSQL are failing in Drone. Unfortunately due to historic reasons we have that difference in the base.sql files for MySQL and PostgreSQL at this point.

Reply:
thanks